src/main/java/duke/tasks/FindCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.tasks;
+
+import java.io.IOException;
+
+/**
+ * Represents a Find Command. This command handles the find input
+ * from users.
+ */
+public class FindCommand extends Command {
+    protected String toFind;
```

Comment:
May I know why the variable accessibility is protected? Is it possible to make it private instead?

Reply:
Also, the naming of this variable may be changed to something like "taskToFind" to make it more informative.